django/db/backends/sqlite3/features.py

```diff
@@ -51,6 +47,16 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_order_by_nulls_modifier = Database.sqlite_version_info >= (3, 30, 0)
     order_by_nulls_first = True
 
+    @cached_property
+    def introspected_field_types(self):
+        return{
+            **super().introspected_field_types,
+            'BigAutoField': 'AutoField',
+            'DecimalField': 'FloatField',
```

Comment:
Based on the test, SQLite introspects DecimalField as DecimalField but with guessed arguments. I would remove this line and replace `if connection.features.can_introspect_decimal_field:` with `if connection.vendor != 'sqlite':`.

Reply:
Thanks. Done!